sycl/test-e2e/Graph/event_profiling_info.cpp

```diff
@@ -130,8 +130,12 @@ int main() {
 
     KernelGraph.end_recording(Queue);
 
-    auto CopyGraphExec = CopyGraph.finalize();
-    auto KernelGraphExec = KernelGraph.finalize();
+    // The profiling is not available with the in-order optimization.
+    // We therefore disable this optimization.
```

Comment:
This comment is a bit confusing, why are we talking about the in-order optimzation at all here? It seems irrelevant to this test.